data/surveysparrow.mdx

```diff
@@ -0,0 +1,20 @@
+---
+name: SurveySparrow
+description: SurveySparrow is an end-to-end conversational experience management platform that helps in getting a 40% better response rate.
+founded: October 1 2017
+based: Both
+domain: Computer And Related Activities
+logo: surveysparrow.jpeg
```

Comment:
❤️ Thanks for your conribution but there is a tiny problem with the filename.  It's supposed to be `SurveySparrow.png` right?

Reply:
Yes, fixed it now 😃

Reply:
Thank you✨, I appreciate it.
![youre-welcome-pleasure](https://media.tenor.com/QiUtH4YUcocAAAAC/youre-welcome-pleasure.gif)